common/source/docs/common-mavlink-mission-command-messages-mav_cmd.rst

```diff
@@ -3069,6 +3069,8 @@ In the example above, the servo attached to output channel 8 would be
 moved to PWM 1700 (servos generally accept PWM values between 1000 and
 2000).
 
+.. note:: as of firmware versions 4.0 and later, this command can be used on any output no matter how it is configured by its ``SERVOx_FUNCTION`` command
```

Comment:
I think it will now work with any of the pass-through options (i.e. "rc1", "rc2") or "0".

Reply:
done...